com.civicrm.hrjobroles/CRM/Hrjobroles/BAO/HrJobRoles.php

```diff
@@ -58,28 +72,14 @@ public static function buildDbOptions($fieldName) {
     $query = "SELECT cpv.value, cpv.label from civicrm_option_value cpv
               LEFT JOIN civicrm_option_group cpg on cpv.option_group_id = cpg.id
               WHERE cpg.name = %1";
-    $options = array();
+    $options = [];
     $result = CRM_Core_DAO::executeQuery($query, $queryParam);
     while ($result->fetch()) {
-      $options[] =  array( 'value'=>$result->value, 'label'=>strtolower($result->label) );
+      $options[$result->value] = strtolower($result->label);
     }
```

Comment:
So you've changed the output of buildDbOptions to be exactly like the output of buildOptions (except for the use of strlower on the label). Why don't you remove this method and use buildOptions instead?